webapp/views/partials/users.jade

```diff
@@ -1,5 +1,6 @@
 div(class='container')
-    h1 Users page
-
-    p(ng-repeat='user in users') Username: {{user.email}} Password: {{user.password}}
+    h1 Welcome
 
+    p(ng-repeat='user in users')
+        content1 Username: {{user.email}} Password: {{user.password}}
```

Comment:
should actually look like:
p(ng-repeat='user in users').
   Username: {{user.email}} Password: {{user.password}}
   Firstname: {{user.firstname}} Lastname {{user.lastname}}